server.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const fs = require('fs');
+const express = require('express');
+const pg = require('pg');
+const app = express();
+const bodyParser = require('body-parser');
+const PORT = process.env.PORT || 3000;
+
+const conString = 'postgres://localhost:5432/portfolioServer';
+const client = new pg.Client(conString);
+
```

Comment:
So you don't really need lines 10 and 11 here...looks like you're trying to run Postgres and connect to portfolioServer? All you need is your node server.